src/search/components/dateTime.component.tsx

```diff
@@ -404,45 +413,31 @@ const DateTimeSearch = (props: DateTimeSearchProps): React.ReactElement => {
               onClose={() => setPopupOpen(false)}
               onError={(error) => setDatePickerToDateError(!!error)}
               views={['year', 'month', 'day', 'hours', 'minutes']}
-              OpenPickerButtonProps={{
-                size: 'small',
-                'aria-label': 'to, date-time picker',
-              }}
-              renderDay={(date, selectedDates, pickersDayProps) =>
-                renderExperimentPickerDay(
-                  datePickerFromDate,
-                  experiments,
-                  isDateTimeInExperiment,
-                  date,
-                  selectedDates,
-                  pickersDayProps
-                )
-              }
-              renderInput={(renderProps) => {
-                const error =
-                  (renderProps.error || invalidDateRange) ?? undefined;
-                let helperText = 'Date-time format: yyyy-MM-dd HH:mm';
-                if (invalidDateRange) helperText = 'Invalid date-time range';
+              slots={{
+                day: (pickersDayProps) =>
+                  renderExperimentPickerDay(
+                    datePickerFromDate,
+                    experiments,
+                    isDateTimeInExperiment,
+                    pickersDayProps
+                  ),
 
-                return (
-                  <TextField
-                    {...renderProps}
-                    fullWidth
-                    id="to date-time"
-                    inputProps={{
-                      ...renderProps.inputProps,
-                      placeholder: 'To...',
-                      'aria-label': 'to, date-time input',
-                      sx: {
-                        fontSize: '1rem',
-                        fontWeight: 'bold',
-                      },
-                    }}
-                    variant="standard"
-                    error={error}
-                    {...(error && { helperText: helperText })}
-                  />
-                );
+                textField: CustomTextField,
+              }}
+              slotProps={{
+                actionBar: { actions: ['clear', 'today'] },
```

Comment:
Again, the `to` pickers have `clear` and `today` whereas the from pickers only have `clear`. I know `today` (i.e. now) is a less useful option for the from picker, but I think it makes sense for them to match